app/models/concerns/graphql_node.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module GraphQLNode
+  extend ActiveSupport::Concern
+
+  def global_relay_id
+    Base64.strict_encode64(["0", self.class.name, ":", id.to_s].join)
```

Comment:
What does the `0` in the front represent?

Reply:
It represents `global_relay_id` format version 0. When decoding these ID's, we assert we're working with version 0 IDs. If we need to change the format of the ID in the future, we'd bump the version number in the new IDs.